lib/bolt/inventory/group2.rb

```diff
@@ -307,21 +313,22 @@ def validate(used_names = Set.new, target_names = Set.new, aliased = {}, depth =
 
         # Collect target names and aliases into a list used to validate that subgroups don't conflict.
         # Used names validate that previously used group names don't conflict with new target names/aliases.
-        @targets.each_key do |n|
+        @targets.each do |t_name, t_data|
           # Require targets to be parseable as a Target.
           begin
-            Target.new(n)
+            # For now just validate that uri can parse, other validation?
```

Comment:
If there is it seems like it should go in `parse_uri`, but I think that validation is ok